src/commands/ban_commands.py

```diff
@@ -21,10 +21,19 @@ async def ban(
         reason: str,
     ):
         """Ban the specified user."""
-        async with create_response_context(interaction) as response_message:
-            async with create_logging_embed(
-                interaction, user=user, reason=reason
-            ) as logging_embed:
-                await ban_user(logging_embed, user, reason)
+        # result is a tuple of state and descriptio
+        result = await ban_user(user, reason)
 
-                response_message.set_string(f"Successfully banned {user.mention}")
+        if result[0]:  # ban succeeded dont create embed.
```

Comment:
```suggestion
        if result[0]:  # ban succeeded
```
i think it's supposed to be like this? i'm kind of confused